package.json

```diff
@@ -48,7 +48,9 @@
     "Marcos Peebles",
     "James Nash (http://cirrus.twiddles.com/)",
     "Valerio Francescangeli",
-    "Daniel Bull"
+    "Daniel Bull",
+    "Daniel Wilkinson",
+    "Charles Brindle (Ash Blue)"
```

Comment:
Are brackets reserved for URLs in the `contributors` section of package.json?
Something somewhere might convert Ash Blue to a hyperlink 🤔

Reply:
Good spot. You're right! https://docs.npmjs.com/files/package.json#people-fields-author-contributors

@marcospeebles Perhaps change this to something like `"Charles \"Ash Blue\" Brindle"`?